src/internal/document_api/mod.rs

```diff
@@ -1330,7 +1330,7 @@ mod tests {
 
         // let's get the policy again, but if the policy future executes (cache miss) error
         get_cached_policy_or(&config, &policy_grant, &policy_cache, async {
-            Err(IronOxideErr::InitializeError)
+            Err(IronOxideErr::InitializeError("".into()))
```

Comment:
Seems like `"SDK initialization failed. Underlying cause ''"` will be confusing. Maybe `InitializeError` could take `Option<String>` so we can differentiate when we want the generic error message vs. a specific one?

Reply:
This is a test. In all real instances there will be an underlying cause.

Reply:
I missed that. Great